src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java

```diff
@@ -156,7 +156,7 @@ private ModelAndView studentSummary(String sectionId, String studentId, boolean
 
     private void addAssignmentSummaryToPage(ModelAndView page, int totalPresentDays, int totalTardyDays, int totalAbsentDays, int totalExcusedDays, int totalDays, AttendanceAssignment assignment) {
 
-        if (assignment != null && totalDays != 0) {
+        if (assignment != null && totalDays != 0 && assignment.getPresentPoints() != null) {
```

Comment:
It's probably safe to check for a blank string with StringUtils. If somebody makes presentPoints default to "" (which is good practice) this check will fail.